packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -283,8 +283,8 @@ class CupertinoTextField extends StatefulWidget {
     this.cursorRadius = const Radius.circular(2.0),
     this.cursorOpacityAnimates = true,
     this.cursorColor,
-    this.selectionHeightStyle = ui.BoxHeightStyle.tight,
-    this.selectionWidthStyle = ui.BoxWidthStyle.tight,
+    this.selectionHeightStyle,
+    this.selectionWidthStyle,
```

Comment:
Should you update the `CupertinoTextField.borderless` constructor below as well? It also has `selectionHeightStyle` and `selectionWidthStyle` arguments with default values

Reply:
Good catch! Updated the borderless constructor as well.